flux/README.md

```diff
@@ -0,0 +1,4 @@
+# Flux
+Flux is most useful when used as a deployment tool at the end of a Continuous Delivery pipeline. Flux will make sure that your new container images and config changes are propagated to the cluster.
+
+> Head over to default Flux [Repository](https://github.com/fluxcd/flux)
```

Comment:
It also uses:
- https://github.com/fluxcd/helm-operator
- https://github.com/fluxcd/flux-recv

Reply:
I kept it short, but you can add those, Will be useful.

Reply:
Can you add them as part of this PR? or remove the readme altogether?

Reply:
I added in a new way, let me know if I should change the way they added.